frontend/packages/ux-editor/src/components/config/editModal/EditOption/EditOption.test.tsx

```diff
@@ -69,6 +69,7 @@ describe('EditOption', () => {
     const fieldset = screen.getByRole('group', { name: legend });
     const textField = within(fieldset).getByRole('textbox', { name: textMock('general.value') });
     await act(() => user.type(textField, 'a'));
+    await act(() => user.click(document.body));
     expect(onChange).toHaveBeenCalledWith({ label, value: value + 'a' });
```

Comment:
This is good, but to prevent the bug from occurring in the future (closing the value field with just one character typed), adding extra characters to the test could be a good idea. What are your thoughts? :)

```suggestion
    await act(() => user.type(textField, 'abc'));
    await act(() => user.click(document.body));
    expect(onChange).toHaveBeenCalledWith({ label, value: value + 'abc' });
```

Reply:
I will fix it 😊